staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -116,8 +119,48 @@ service RuntimeService {
     rpc Status(StatusRequest) returns (StatusResponse) {}
 }
 
+message GetCRIEventsRequest {
+}
+
+message CRIEventsResponse {
+    // ID of the container/sandbox
+    string container_id = 1;
+
+    // Type of the cri event
+    CRIEventType cri_event_type = 2;
+
+    // Creation timestamp of this event
+    int64 created_at = 3;
+
+    // source of the event
+    CRIEventSource cri_event_source = 4;
+
+    // Sandbox metadata
+    PodSandboxMetadata pod_sandbox_metadata = 5;
+}
+
+enum CRIEventSource {
+    CRI_CONTAINER = 0;
+    CRI_SANDBOX = 1;
```

Comment:
if the container is the sandbox itself..we normally filter these unless there is a request to include..